scrubber.js

```diff
@@ -130,6 +133,8 @@ ScrubberView.prototype.attachListeners = function ()  {
   };
 
   var stop = function () {
+    document.removeEventListener('mouseup', stop, true);
+    document.removeEventListener('touchend', stop, true);
```

Comment:
If I understand correctly, using the third "useCapture" parameter here is not desirable. Ref:

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener

> Specifies whether the EventListener to be removed is registered as a capturing listener or not. If this parameter is absent, a default value of false is assumed.
>
> If a listener is registered twice, one with capture and one without, remove each one separately. Removal of a capturing listener does not affect a non-capturing version of the same listener, and vice versa.

So I think we'd only want to set the "useCapture" parameter to `true` here if we had also set it to `true` when we registered the event listener.

Reply:
Ah, that makes sense. I was on that page earlier today, but missed that. I'll change and resubmit. Thanks!